src/main/java/org/folio/inventory/dataimport/util/AdditionalFieldsUtil.java

```diff
@@ -376,17 +376,24 @@ private static String normalizeContent(Object content) {
    *
    * @param record    record that needs to be updated
    * @param fieldName tag of the field
+   * @param subfield  subfield of the field
+   * @param value     value of the field
    * @return true if succeeded, false otherwise
    */
-  public static boolean removeField(Record record, String fieldName) {
+  public static boolean removeField(Record record, String fieldName, char subfield, String value) {
```

Comment:
@Maksat-Galymzhan  Do we have separate tests for this class? Please add these specific scenarios for the AdditionalFieldsUtil test scenarios. This is a significant change in our MARC-logic manipulation and I think it should be covered with unit tests. Many thanks for considering my request!

Reply:
@Maksat-Galymzhan  Great work! I found that you added new tests here. Thank you. Closed this thread.